terms-conditions/Certified_Kubernetes_Terms.md

```diff
@@ -64,7 +64,7 @@ _Conformance Time Period; Later Versions of Kubernetes_. A Participant’s permi
 * 12 months after the release date of Kubernetes x.y, or
 * 9 months after the release date of the next minor release (e.g., Kubernetes x.y+1).
 
-However, notwithstanding the time limits above, the Participant may also continue to use the Certified Kubernetes Marks and Participant Kubernetes Combinations for any prior version of Kubernetes for which the offering was a Qualifying Offering, for so long as (1) the offering continues to be a Qualifying Offering for each subsequent version of Kubernetes; (2) the Qualifying Offering with the subsequent version of Kubernetes is made generally available to users of the prior version; and (3) the Participant otherwise continues to abide by the terms of the Conformance Program.
+However, notwithstanding the time limits above, the Participant may also continue to certify and use the Certified Kubernetes Marks and Participant Kubernetes Combinations for any prior version of Kubernetes for which the offering was a Qualifying Offering, for so long as (1) the offering continues to be a Qualifying Offering for each subsequent version of Kubernetes; (2) the Qualifying Offering with the subsequent version of Kubernetes is made generally available to users of the prior version; and (3) the Participant otherwise continues to abide by the terms of the Conformance Program.

```

Comment:
Just FYI - this diff not showing what changed is why some groups try to stick with 80 columns  :-)
Is the only diff the new words "certify and" in the first line?